### PR TITLE
More restrictive license

### PR DESCRIPTION
**Pull request by okonet**
In order to prevent others from taking materials and running similar workshop, the license should be changed to something more restrictive.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -67,7 +67,9 @@ Or:
 
 [Artem Sapegin](http://sapegin.me), [Andrey Okonetchnikov](http://okonet.ru/) and [contributors](https://github.com/component-driven/component-driven-development/graphs/contributors).
 
-MIT License, see the included [License.md](License.md) file.
+This material is available for private, non-commercial use under the GPL version 3. If you would like to use this material to conduct your own workshop, please contact me at hi@component-driven.io
```
Comment (sapegin) [inline Readme.md]:
```suggestion
This material is available for private, non-commercial use under the GPL version 3. If you would like to use this material to conduct your own workshop, please contact us at hi@component-driven.io.
```

**Review by sapegin — APPROVED** [review]
I think it's worth adding a file with the text of the license.

**Comment by okonet** [comment]
@sapegin thought the same but then realized it makes it harder to maintain. Having a reference to the license in Readme and `package.json` should be enough, no?

**Comment by okonet** [comment]
I'll merge this and we can add the license later if we decide to.

**Pull request merged**